common/v2/components/WalletUnlock/Web3ProviderInstall.tsx

```diff
@@ -25,34 +25,44 @@ function InstallTrunk() {
           <div className="TrustWallet-container">
             <NewTabLink href="https://trustwallet.com/dapp">
               <div className="TrustWallet-img">
-                <img src={TrustWalletWEBP} />
+                <img src={TrustWalletSVG} />
               </div>
             </NewTabLink>
 
             <Typography>{translateRaw('TRUST_APP_LABEL')}</Typography>
-            <NewTabLink href="https://itunes.apple.com/us/app/trust-ethereum-wallet/id1288339409">
+            <NewTabLink
+              className="download-option"
+              href="https://itunes.apple.com/us/app/trust-ethereum-wallet/id1288339409"
+            >
               <img src={AppStoreBadgeIMG} />
             </NewTabLink>
-            <NewTabLink href="https://play.google.com/store/apps/details?id=com.wallet.crypto.trustapp">
+            <NewTabLink
+              className="download-option"
+              href="https://play.google.com/store/apps/details?id=com.wallet.crypto.trustapp"
+            >
               <img src={GooglePlayBadgeIMG} />
             </NewTabLink>
           </div>
```

Comment:
Can we refactor this please to use `config/wallets.ts` ?
- having the download links in the config file can helps us be more careful about url changes
we can add a param like:
```
  install: {
    getItLink: ?
    google: 'https://play.google.com/<app_id>'
  }
```
- icons are already available in the config file (so we update them in a single location)
- the translation labels as well (e.g lid)
- once we extract the data into the config file we can reduce the `tsx` by iterating on the list of valid wallets.

Reply:
I'm not sure I agree with this. Since we show Coinbase Wallet etc, which we don't have as a wallet in `config/wallets.ts``. We shouldn't be adding extra wallets just to store their download links imo.

Reply:
Unless we want to support Coinbase Wallet with an icon etc in WalletList

Reply:
It seems like we already do: L50 `<img src={CoinbaseWalletJPG} />`
I find it useful to have a single place to discover all the wallets we support as well as a single make changes.